packages/espresso/README.md

```diff
@@ -0,0 +1,121 @@
+# espresso
+
+Provides bindings for Espresso tests of Flutter Android apps.
+
+## Installation
+
+Add ```android:usesCleartextTraffic="true"``` in the ```<application>``` in the AndroidManifest.xml
+of the Android app used for testing.
```

Comment:
Perhaps add a note for "don't forget to turn this off before you ship"?

Reply:
Can we create a separate testing manifest file? Internally, we encourage developers to use different manifest files in this case: one for the production app, the other for testing purpose only, so they won't need to worry about leaking testing only permissions/configurations to their production app.

Reply:
Sure thing, I'll do that in the example app.

Reply:
done, and updated the README to mention it